content/en/docs/tasks/administer-cluster/declare-network-policy.md

```diff
@@ -97,14 +96,21 @@ metadata:
 spec:
   podSelector:
     matchLabels:
-      run: nginx
+      app: nginx
   ingress:
   - from:
     - podSelector:
         matchLabels:
           access: "true"
 ```
 
+{{< note >}}
+
+In the case, the label `app=nginx` is automatically added.
```

Comment:
@diguage I'm not clear on what you wanted the reader to learn here. It might be OK to leave that note out.

Reply:
Yes. If the note does not exist, the example is OK. But it is much clearer with the note.
Sorry to have kept you waiting. I'm on holiday.